test_tigervnc.py

```diff
@@ -18,23 +19,24 @@
 def test_tigervnc_config():
     vncviewercmd = kvl.config.get('vncviewer', 'vncviewer')
 
-    try:
-        cmd = [vncviewercmd, '--help']
-        kvl.log.info(f'Checking VNC viewer: {" ".join(cmd)}')
-        help_STDOUT = subprocess.check_output(cmd)
-    except:
-        kvl.log.warning(f'Command {" ".join(cmd)} Failed')
-        raise
+    cmd = [vncviewercmd, '--help']
+    kvl.log.info(f'Checking VNC viewer: {" ".join(cmd)}')
+    result = subprocess.run(cmd, capture_output=True)
+    output = result.stdout.decode() + '\n' + result.stderr.decode()
 
-    if re.search(r'TigerVNC', help_STDOUT.decode()):
+    if re.search(r'TigerVNC', output):
         # VNC Viewer is TigerVNC, so we need to verify the RemoteResize config
-        with open(Path('~/.vnc/default.tigervnc').expanduser()) as FO:
+        tigervnc_config_file = Path('~/.vnc/default.tigervnc').expanduser()
+        assert tigervnc_config_file.exists()
```

Comment:
When this fails you don't get a concise message as to why in the pytest output.  I think a log.error here might be good like the others below.